homeassistant/components/stream/worker.py

```diff
@@ -61,12 +35,51 @@ def __init__(self, outputs_callback) -> None:
         self._outputs: list[StreamOutput] = []
         self._sequence = 0
         self._segment_start_pts = None
-        self._stream_buffer = None
+        self._memory_file: BytesIO = cast(BytesIO, None)
+        self._av_output: av.container.OutputContainer = None
+        self._segment: Segment = cast(Segment, None)
+        self._input_video_stream: av.video.VideoStream = None
+        self._input_audio_stream = None  # type=Optional[av.audio.AudioStream]
```

Comment:
Please use proper type annotations 🙂
```suggestion
        self._input_audio_stream: av.audio.AudioStream | None = None
```

Reply:
Please let's not worry about type hints in these intermediate PRs. We can go back and fix it up after everything is done or you can make suggestions on the main PR. Otherwise it's a lot of busy work and @allenporter already said it was making it hard to follow.

Reply:
It's hard to follow on that PR which is huge, this is one is pretty small 🙂

If it's not trivial to fix, then sure, it can be done in subsequent PR.

Reply:
I will end up making the change, but as you can see the old style type hint is just copied straight from the deleted line so it actually makes it easier to follow as is. Plus it's just a comment because of the known issue with pyav audio stream types